### PR TITLE
Docs: Update Hive doc page with the 4.0.0-alpha-1 features

### PR DESCRIPTION
**Pull request by samredai**
Original PR -> PR #4915

Co-authored-by: pvary pvary@cloudera.com

### CODE REVIEW

**Review by pvary — APPROVED** [review]

**Comment by pvary** [comment]
Merged to master.
Thanks @samredai for the rebase

File: docs/hive.md [hunk 1]
```diff
@@ -24,20 +24,63 @@ menu: main
...
+Hive 4.0.0-alpha-1 comes with the Iceberg 0.13.1 included. No additional downloads or jars are needed.
+
+### Hive 2.3.x, Hive 3.1.x
+
+You need to do the following steps:
+* Loading runtime jar
+* Enabling support
+Collaborator
+@samredai samredai 26 days ago
+This could just be one line right?
```
Comment (openinx) [inline docs/hive.md:75]: @samredai @pvary , is this correct ?  are we just adding the unintentional comments here ?

File: docs/hive.md [hunk 2]
```diff
@@ -24,20 +24,63 @@ menu: main
...
+
+In order to use Hive 2.3.x or Hive 3.1.x, you must load the Iceberg-Hive runtime jar and enable Iceberg support, either globally or for an individual table using a table property.
+
+and then jump into the sections below that provide more details.
+
+@samredai	Reply...
```
Comment (openinx) [inline docs/hive.md:81]: Here is the similar issue ?

File: docs/hive.md (hunk 1 above)
Comment (samredai) [reply docs/hive.md:75]: Ah thanks for catching this, let me open a PR

Comment (samredai) [reply docs/hive.md:75]: https://github.com/apache/iceberg/pull/5202

**Pull request merged**